wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -74,7 +74,7 @@ public final class Constants {
                 IS_PROD_BUILD = true;
                 FILENAME_NETWORK_SUFFIX = "";
                 WALLET_NAME_CURRENCY_CODE = "dash";
-                org.dash.wallet.common.util.Constants.INSTANCE.setEXPLORE_GC_FILE_PATH("explore/explore.db");
+                org.dash.wallet.common.util.Constants.INSTANCE.setEXPLORE_GC_FILE_PATH("explore/explore-devnet.db");
```

Comment:
The same concern here as before. Can we not use testnet builds to test this database and not modify production values?

Reply:
This is where it gets tricky.  DashDirect doesn't support testnet, only mainnet.  Purchases couldn't happen, but other aspects of the new feature could be tested.

Additionally, the production and testnet databases are version 1.  

`explore-devnet.db` is version 2.  I was thinking about renaming this to `explore-v2.db` which also needs to be the name created by **Explore Sync Tool**.  Version 2 not only has extra fields, but it also has different data since it comes from the devapi server where `merchantId`'s differ and the deeplinks are currently missing.  So we can't use testnet builds, but need to make devapi builds (it is named devnet, but it has nothing to do with any devnets).